dist/taginfo.json

```diff
@@ -43,86 +63,362 @@
+    },
+    {
+      "key": "amenity",
+      "value": "kindergarten",
+      "doc_url": "https://nsi.guide/?t=operators&k=amenity&v=kindergarten"
+    },
+    {
+      "key": "amenity",
+      "value": "language_school",
+      "doc_url": "https://nsi.guide/?t=brands&k=amenity&v=language_school"
+    },
+    {
+      "key": "amenity",
+      "value": "library",
+      "doc_url": "https://nsi.guide/?t=operators&k=amenity&v=library"
+    },
+    {
+      "key": "amenity",
+      "value": "marketplace",
+      "doc_url": "https://nsi.guide/?t=operators&k=amenity&v=marketplace"
+    },
+    {
+      "key": "amenity",
+      "value": "money_transfer",
+      "doc_url": "https://nsi.guide/?t=brands&k=amenity&v=money_transfer"
+    },
+    {
+      "key": "amenity",
+      "value": "mortuary",
+      "doc_url": "https://nsi.guide/?t=brands&k=amenity&v=mortuary"
+    },
+    {
+      "key": "amenity",
+      "value": "motorcycle_rental",
+      "doc_url": "https://nsi.guide/?t=brands&k=amenity&v=motorcycle_rental"
+    },
+    {
+      "key": "amenity",
+      "value": "music_school",
+      "doc_url": "https://nsi.guide/?t=brands&k=amenity&v=music_school"
+    },
+    {
+      "key": "amenity",
+      "value": "parcel_locker",
+      "doc_url": "https://nsi.guide/?t=brands&k=amenity&v=parcel_locker"
+    },
+    {
+      "key": "amenity",
+      "value": "parking",
+      "doc_url": "https://nsi.guide/?t=operators&k=amenity&v=parking"
+    },
+    {
+      "key": "amenity",
+      "value": "payment_centre",
+      "doc_url": "https://nsi.guide/?t=brands&k=amenity&v=payment_centre"
+    },
+    {
+      "key": "amenity",
+      "value": "payment_terminal",
+      "doc_url": "https://nsi.guide/?t=brands&k=amenity&v=payment_terminal"
+    },
+    {
+      "key": "amenity",
+      "value": "pharmacy",
+      "doc_url": "https://nsi.guide/?t=brands&k=amenity&v=pharmacy"
+    },
+    {
+      "key": "amenity",
+      "value": "photo_booth",
+      "doc_url": "https://nsi.guide/?t=brands&k=amenity&v=photo_booth"
+    },
+    {
+      "key": "amenity",
+      "value": "police",
+      "doc_url": "https://nsi.guide/?t=operators&k=amenity&v=police"
+    },
+    {
+      "key": "amenity",
+      "value": "post_box",
+      "doc_url": "https://nsi.guide/?t=operators&k=amenity&v=post_box"
+    },
+    {
+      "key": "amenity",
+      "value": "post_depot",
+      "doc_url": "https://nsi.guide/?t=operators&k=amenity&v=post_depot"
+    },
+    {
+      "key": "amenity",
+      "value": "post_office",
+      "doc_url": "https://nsi.guide/?t=operators&k=amenity&v=post_office"
+    },
+    {
+      "key": "amenity",
+      "value": "prep_school",
+      "doc_url": "https://nsi.guide/?t=brands&k=amenity&v=prep_school"
+    },
+    {
+      "key": "amenity",
+      "value": "prison",
+      "doc_url": "https://nsi.guide/?t=operators&k=amenity&v=prison"
+    },
+    {"key": "amenity", "value": "pub", "doc_url": "https://nsi.guide/?t=operators&k=amenity&v=pub"},
```

Comment:
What decides which tag of operators and brands gets picked? There are more brands (58) than operators (17) yet the link goes to the latter? Assuming you can't put two links in, or two entries, maybe linking to the bigger one, or defaulting to brand if present or something might make sense?